src/components/RengoManagementPane/RengoManagementPane.tsx

```diff
@@ -15,6 +15,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { interpolate } from "goban";
```

Comment:
I think this needs to come from the translate module.  No sure what happens if you use goban's version...

Reply:
LOL VSCode put that there for me.  It "seems" to work   :D